nano/core_test/bootstrap_ascending.cpp

```diff
@@ -135,8 +135,13 @@ TEST (account_sets, priority_down_sat)
 	auto store = nano::make_store (system.logger, nano::unique_path (), nano::dev::constants);
 	ASSERT_FALSE (store->init_error ());
 	nano::bootstrap_ascending::account_sets sets{ system.stats };
-	sets.priority_down (account);
-	ASSERT_EQ (1.0f, sets.priority (account));
+	sets.priority_up (account);
```

Comment:
Can we check the return value here so it's checked at least somewhere. It looks like it's just debugging code for now though it may be used eventually.